contracts/stellar-interchain-token-service/src/token_handler.rs

```diff
@@ -45,6 +52,9 @@ pub fn give_token(
         TokenManagerType::NativeInterchainToken => {
             token_manager.mint_from(env, &token_address, recipient, amount)
         }
+        TokenManagerType::MintBurnFrom => {
+            token_manager.mint_from(env, &token_address, recipient, amount)
+        }
```

Comment:
```suggestion
        TokenManagerType::NativeInterchainToken | TokenManagerType::MintBurnFrom => {
            token_manager.mint_from(env, &token_address, recipient, amount)
        }
```
reuse match guard to avoid duplication